src/py/aspen/database/models.py

```diff
@@ -26,6 +26,8 @@ class Group(base):
     __tablename__ = "groups"
 
     name = Column(String, unique=True, nullable=False)
+    gisaid_division_name = Column(String, unique=True)
+    email = Column(String, unique=True, nullable=False)
```

Comment:
So the reason is that public archives seem to want an email address.  In that case, whose email address should we use?  It would seem that a group email address is appropriate here.

Reply:
ok makes sense.